Assets/MixedRealityToolkit.SDK/Features/UX/Scripts/Pointers/TeleportPointer.cs

```diff
@@ -362,7 +362,11 @@ public override void OnInputChanged(InputEventData<Vector2> eventData)
                             {
                                 canMove = false;
                                 // Rotate the camera by the rotation amount.  If our angle is positive then rotate in the positive direction, otherwise in the opposite direction.
-                                MixedRealityToolkit.Instance.MixedRealityPlayspace.RotateAround(CameraCache.Main.transform.position, Vector3.up, angle >= 0.0f ? rotationAmount : -rotationAmount);
+                                MixedRealityPlayspace.PerformTransformation(
+                                    p =>
+                                    {
+                                        p.RotateAround(CameraCache.Main.transform.position, Vector3.up, angle >= 0.0f ? rotationAmount : -rotationAmount);
+                                    });
```

Comment:
Is OnInputChanged called in the update loop (i.e. every frame?) If so we should avoid using lambdas here because they contribute to frame-by-frame memory alloc (and then eventually GC thrashing)

Reply:
good question. it might be when the pointer is displayed. 

tbh, i adapted this code from a previous PR that used the lambda.